examples/trusted_information_retrieval/module_0/rust/src/tests.rs

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::config::parse_config_file;
+use assert_matches::assert_matches;
+
+const CONFIG_FILE: &str = r#"
+database_url = "https://localhost:8888"
+"#;
+
+#[test]
+fn test_parse_config() {
+    let config = parse_config_file(CONFIG_FILE.as_bytes().to_vec());
+    assert_matches!(config, Ok(_));
```

Comment:
It seems you know exactly what the result should be, may as well inline everything.

```
let parsed = ...
let expected = Database {
  ...
};
assert_eq!(Ok(expected)), parsed)
```